addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -283,6 +285,53 @@ var Wysiwyg = Widget.extend({
             mediaDialog.on('cancel', this, resolve);
         });
     },
+    changeSelectedTextColor(color, closeColorPicker = false) {
+        if(color === "") {
+            this.editor.execCommand('uncolorText');
+            $(window).trigger('click');
+            return;
+        }
+
+        if(this.colorPicker.colorNames.indexOf(color) !== -1) {
+           color = "var(--" + color + ")";
+        }
+        this.editor.execCommand('colorText', {color: color});
+
+        if(closeColorPicker) {
+            $(window).trigger('click');
+        }
+    },
+    toggleColorPicker() {
+        return new Promise((resolve)=> {
+            if(this.colorPicker) {
+                $(window).trigger('click');
+                return;
+            }
+            const self = this;
+            this.colorPicker = new ColorPaletteWidget(this, {});
+            this.colorPickerContainer = $('<div class="o_colorpicker_container"></div>');
+            this.colorPicker.appendTo(this.colorPickerContainer);
+
+            this.colorPicker.on('custom_color_picked', this,  async (e) => {
```

Comment:
Why are you using the keywoard `async` if you don't use `await`?

Reply:
True, it's a relic I forgot to remove